### PR TITLE
add a post about rack-protection v1.5.5

### PR DESCRIPTION
**Pull request by namusyaka**
I've just add a blog post for v1.5.5.
/cc @zzak @kgrz @olleolleolle

### CODE REVIEW

File: _posts/2018-03-08-rack-protection-1.5.5.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+publish_date: Thursday, March 8, 2018
+---
+
+I have just released Rack Protection v1.5.5 for backporting security fix.
+
+## Details
+
+The v1.5.5 contains a security fix for [CVE-2018-1000119](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-1000119).
+
+It was determined a timing attack vulnerability in the CSRF token checking that can result in signatures can be exposed.
```
Comment (olleolleolle) [inline _posts/2018-03-08-rack-protection-1.5.5.markdown]: `can be exposed` => `being exposed`.

File: _posts/2018-03-08-rack-protection-1.5.5.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+[The original fix](https://github.com/sinatra/sinatra/commit/8aa6c42ef724f93ae309fb7c5668e19ad547eceb) has already been merged at rack-protection v2.0.0.rc3. Therefore, there is no problem if you are using rack-protection v2.0.0.rc3 or later.
+
+If you still are using rack-protection v1.5.X, we would highly recommend to upgrade the gem.
+
+## Thank you
+
+Finally, I am deeply grateful to [Andreas Karlsson](https://github.com/jeltz) and [Kurt Seifried](https://github.com/kseifriedredhat) who worked on this issue.
+Thank you.
```
Comment (olleolleolle) [inline _posts/2018-03-08-rack-protection-1.5.5.markdown]: This is a nice section!

File: _posts/2018-03-08-rack-protection-1.5.5.markdown (hunk 1 above)
Comment (namusyaka) [reply _posts/2018-03-08-rack-protection-1.5.5.markdown]: done: https://github.com/sinatra/sinatra.github.com/pull/235/commits/915d2520bc324bcdef49a3661a32049c555577d3

**Comment by namusyaka** [comment]
I'm going to wait until @zzak reviews this post.

File: _posts/2018-03-08-rack-protection-1.5.5.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: Rack Protection v1.5.5 is out
```
Comment (zzak) [inline _posts/2018-03-08-rack-protection-1.5.5.markdown]: I've been using the fully qualified module name when referencing it, like Rack::Protection or `rack-protection` when referring to the gem.

File: _posts/2018-03-08-rack-protection-1.5.5.markdown [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+## Details
+
+The v1.5.5 contains a security fix for [CVE-2018-1000119](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-1000119).
+
+It was determined a timing attack vulnerability in the CSRF token checking that can result in signatures being exposed.
+
+[The original fix](https://github.com/sinatra/sinatra/commit/8aa6c42ef724f93ae309fb7c5668e19ad547eceb) has already been merged at rack-protection v2.0.0.rc3. Therefore, there is no problem if you are using rack-protection v2.0.0.rc3 or later.
+
+If you still are using rack-protection v1.5.X, we would highly recommend to upgrade the gem.
```
Comment (zzak) [inline _posts/2018-03-08-rack-protection-1.5.5.markdown]: A lot of people don't realize they're depending on this gem, or the version, so I'd suggest people check their `Gemfile.lock` to ensure their on the latest version, or specifically add `<insert-gem-dep-line> here to Gemfile`.

This came up here too:
https://github.com/sinatra/sinatra/issues/1408

P.s. maybe it should be `gem "rack-protection", "~> 1.5"` but I'm not sure that will break stuff when people try to upgrade that needs 2.0+. But maybe better than specificly `1.5.5` in there, up to you :raised_hands:

Comment (namusyaka) [reply _posts/2018-03-08-rack-protection-1.5.5.markdown]: I've just updated description: https://github.com/sinatra/sinatra.github.com/pull/235/commits/6b46e52821dcef1e66e4a1ecd2e54e904a2f2a06
PTAL.

File: _posts/2018-03-08-rack-protection-1.5.5.markdown (hunk 3 above)
Comment (namusyaka) [reply _posts/2018-03-08-rack-protection-1.5.5.markdown]: done: https://github.com/sinatra/sinatra.github.com/pull/235/commits/6b46e52821dcef1e66e4a1ecd2e54e904a2f2a06

**Comment by zzak** [comment]
LGTM, merging

**Pull request merged**